docker/run.sh

```diff
@@ -15,20 +15,17 @@ if [ $# -eq 2 ]; then
   fi
 fi
 
-
-# build docker container
-npm install & npm run build
+# build the image
 docker build -f docker/Dockerfile . --tag=vectara-answer:latest
-
 # run the docker with the right configuration
 docker rm vanswer -f 2> /dev/null
 
 # If we used legacy configs (prepared via Python and using queries JSON file),
 # use the Docker command to forward the queries.json path to the container.
 if [ -z "$correct_path" ]; then
-  docker run --platform=linux/amd64 -d --env-file .env --name vanswer -p 127.0.0.1:80:3000/tcp vectara-answer
+  docker run -d --env-file .env --name vanswer -p 80:3000/tcp vectara-answer
```

Comment:
Why did you take out the platform? This is important to keep in, as if you do not and run e.g. on Mac it generates the docker using ARM architecture which is not what we want (may create issue). Can we please put that back in?

Reply:
I have removed the platform arg because it is already specified in the Dockerfile. I think that is enough. Let me know what you think 🤔.

Reply:
Ah, good point. Thank you - yes I think that should work.
Will test locally later today.

Reply:
Okay :+1:.

Reply:
@ofermend Did this work?